_std/defines/mining_z_gen.dm

```diff
@@ -0,0 +1,19 @@
+#define AST_MINSIZE 7        //Min range before rng kicks in
```

Comment:
this isn't just mining z, prob. better named `worldgen.dm`

Reply:
alright ty